CHANGELOG.md

```diff
@@ -23,7 +69,10 @@ See the [README](https://github.com/broadinstitute/cromwell#get-apiworkflowsvers
 * The workflow submission parameters `wdlSource` and `wdlDependencies` have been deprecated in favor of `workflowSource` and
 `workflowDependencies` respectively.  The older names are still supported in Cromwell 28 with deprecation warnings but will
 be removed in a future version of Cromwell.
+
+### Labels
 * A new `/labels` endpoint has been added to update labels for an existing workflow. See the [README](README.md#patch-apiworkflowsversionidlabels) for more information.
+* Label formatting requirements have been updated, please check the [README](README.md#label-format) for more detailed documentation.
```

Comment:
The rules for label values now have more room for variation (i.e, they can be empty strings or start with a number). Whatever custom labels someone was submitting previously should still be valid.